tests/devtools/useAtomDevtools.test.tsx

```diff
@@ -98,22 +96,8 @@ describe('If there is no extension installed...', () => {
       'Please install/enable Redux devtools extension'
     )
 
-    process.env.NODE_ENV = originalNodeEnv
     console.warn = originalConsoleWarn
   })
-
-  it('does not warn if not in dev env', () => {
-    const consoleWarn = jest.spyOn(console, 'warn')
-
-    render(
-      <Provider>
-        <Counter />
-      </Provider>
-    )
-    expect(consoleWarn).not.toBeCalled()
-
-    consoleWarn.mockRestore()
```

Comment:
Thoughts: it could be worth setting up a test matrix variant to test with the different build outputs. Or at least running jest with `__DEV__=false` as well — we should cover production behavior!

Reply:
Let me see if we can setup the test matrix in the CI.